atd-vze/src/helpers/format.js

```diff
@@ -4,4 +4,4 @@ export const formatCostToDollars = cost =>
   !!cost ? `$${cost.toLocaleString()}` : "--";
 
 export const formatDateTimeString = datetime =>
-  format(parseISO(datetime), "yyyy-MM-dd hh:mm:ss a");
+  format(parseISO(datetime), "yyyy-MM-dd h:mm:ss a");
```

Comment:
i don't like using zero-padded timestamps  in the VZE—maybe this is too much of a person preference? The result is that `2024-05-01 05:10:01 am` becomes `2024-05-01 5:10:01 am`. And maybe in a future enhancement we add relative timestamps, Moped style?

Reply:
🙏 i'm all about prettifying/unifying how we display timestamps in this here! ✨ the current format(s?) also stick out to me